src/checkboxFormValidation.ts

```diff
@@ -0,0 +1,10 @@
+export function validateHelpRequest(req, res) {
```

Comment:
Use of a function for validating the field is a good idea.
Now, you need to make it more "generic" because you're doing nothing specific to organisationType and the logic will be the same for every fields/form we have to add.
Use a generic function can help to reduce the amount of code we have to support and also improve the quality.
How to make it more generic? In this case, just by changing the function name and the variables/parameters name.
How to detect if a function can be generic? If you create at least twice the same logic in functions, that means you can make a generic function

Reply:
Yep you're right . I have just pushed the code in which I made the function generic , could you please review. Thank you !